deployment/v3/mosip/README.md

```diff
@@ -1,5 +1,12 @@
 # MOSIP Core
 
+## Prerequisites
+* In case you are using private docker images for MOSIP services
+  * Get ready with below list of details for all the private registries to be used as per [prerequisites](docker-secrets/README.md#Prerequisites)
```

Comment:
Prepare the list of details for all the private registries to be used as per the [pre-requisites].